pkg/build/api/types.go

```diff
@@ -55,6 +55,9 @@ type BuildParameters struct {
 
 	// Output describes the Docker image the Strategy should produce.
 	Output BuildOutput `json:"output,omitempty"`
+
+	// Compute resource requirements to execute the build
+	Resources kapi.ResourceRequirements `json:"resources,omitempty"`
```

Comment:
We should be consistent with upstream.  It's trivial to change, and having it be nil doesn't help us decide user intent.